<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadioBrowser</class>
 <widget class="QDialog" name="RadioBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Radio Browser</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/tomato.png</normaloff>../assets/tomato.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="radioBrowserInput">
       <property name="placeholderText">
        <string>Enter text...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="radioBrowserSearch">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="radioBrowserTree">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="radioBrowserSubmit">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
